docs/pages/modules/config-plugin-and-native-module-tutorial.mdx

```diff
@@ -112,65 +146,70 @@ In another terminal window, compile and run the example app:
+
+Here is an example of a basic config plugin function:
 
 ```js
 const withMyApiKey = config => {
   return config;
 };
 ```
 
-Additionally, you can use `mods`, which are async functions that modify files in native projects such as source code or configuration (plist, xml) files. The `mods` object is different from the rest of the app config because it doesn't get serialized after the initial reading. This means you can use it to perform actions _during_ code generation.
+You can also use `mods`, which are async functions that modify files in native projects, such as source code or configuration files (plist, xml). The `mods` object is different from the rest of the app config because it doesn't serialize after the initial reading. This allows you to perform actions _during_ code generation.
 
-However, there are a few considerations that we should follow when writing config plugins:
+When writing config plugins, follow these considerations:
 
-- Plugins should be synchronous and their return value should be serializable, except for any `mods` that are added.
-- `plugins` are always invoked when the config is read by the `expo/config` method `getConfig`. However, `mods` are only invoked during the "syncing" phase of `npx expo prebuild`.
+- Plugins must be synchronous, and their return value must be serializable, except for any `mods` that are added.
+- `plugins` are invoked whenever the `expo/config` method `getConfig` reads the config. In contrast, `mods` are invoked only during the "syncing" phase of `npx expo prebuild`.
 
-> Although not required, we can use [`expo-module-scripts`](https://www.npmjs.com/package/expo-module-scripts) to make plugin development easier — it provides a recommended default configuration for TypeScript and Jest. For more information, see [config plugins guide](https://github.com/expo/expo/tree/main/packages/expo-module-scripts#-config-plugin).
+> Although optional, use [`expo-module-scripts`](https://www.npmjs.com/package/expo-module-scripts) to simplify plugin development. It provides a recommended default configuration for TypeScript and Jest. For more information, see the [config plugins guide](https://github.com/expo/expo/tree/main/packages/expo-module-scripts#-config-plugin).
 
-Let's start by creating our plugin with this minimal boilerplate. This will create a **plugin** folder where we will write the plugin in TypeScript and add an **app.plugin.js** file in the project root, which will be the entry file for the plugin.
+Start creating your plugin with this minimal boilerplate. Create a **plugin** folder for writing the plugin in TypeScript and add an **app.plugin.js** file in the project root as the plugin's entry point.
 
-1. Create a **plugin/tsconfig.json** file:
+### Create a **plugin/tsconfig.json** file:
```

Comment:
```suggestion
### Create a plugin/tsconfig.json file
```

No affect of having file names bold when the content is rendered. So let's remove them from section headings.